src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/commands.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import cli_command
+from azure.cli.command_modules.consumption._client_factory import usage_details_mgmt_client_factory
+from azure.cli.command_modules.consumption._transformers import transform_usage_list_output
+
+usage_path = 'azure.cli.command_modules.consumption.custom#cli_consumption_list_usage'
```

Comment:
The variable can only be correctly used if it's used once (you wouldn't have two commands pointing to the same method) so I would suggest inlining this and removing the variable.